api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -0,0 +1,235 @@
+  common.Bits synchronization_word = 6;
+
+  // Whether convolutional code descrambling is enabled.
+  bool is_convolutional_code_descrambling = 7;
+
+  // Whether convolutional code decoding is enabled.
+  bool is_convolutional_code_decoding = 8;
+
+  // Whether convolutional codes are inverted.
+  bool is_convolutional_code_inverted = 9;
+
+  // Whether reed solomon coding is being used.
+  bool is_reed_solomon_enabled = 10;
+
+  // The size of a frame, in bits.
+  uint32 frame_size = 11;
+
+  // Whether frame checking is enabled.
+  bool is_frame_checking_enabled = 12;
+}
+
+// The current configuration of a ground station. This is controlled by the parameters of a pass,
+// based on the configuration of the satellite it will communicate with. When debugging issues with
+// a pass, it is good to first confirm that the actual reported configuration matches the expected
+// settings.
+message GroundStationConfiguration {
+
+  // The current configuration of the transmitter in use by the ground station. This configuration
+  // should match the parameters of the executing pass. When debugging issues with data
+  // transmission (e.g., no response from satellite), it can be useful to confirm these values match
+  // the expected configuration of the transmitter.
+  TransmitterConfiguration transmitter = 1;
+
+  // The current configuration of the receiver in use by the ground station. This configuration
+  // should match the parameters of the executing pass. When debugging issues with data
+  // reception (e.g., can't decode signal), it can be useful to confirm these values match
+  // the expected configuration of the receiver.
+  ReceiverConfiguration receiver = 2;
+
+  // The current configuration of the antenna used by the ground station. These are the physical
+  // parameters of an antenna and in practice should never change, even across passes. But natural
+  // events such as earthquakes can cause small changes in these values. When debugging issues with
+  // tracking a satellite (e.g., getting no signal at all), it can be useful to confirm these values
+  // match the expected configuration of the antenna.
+  antenna.AntennaConfiguration antenna = 3;
+}
+
+// The current state of the ground station's transmitter during the operation of a pass.
+message TransmitterState {
+  // The current center frequency of the transmitter, taking into account e.g., Doppler correction.
+  uint64 center_frequency_hz = 1;
+
+  // The current level of the carrier signal, in dBm (decibal-milliwatts).
+  double carrier_level_dbm = 2;
+}
+
+// A current status of convolutional coding.
+enum ConvolutionalCodingStatus {
+  // Not locked on.
+  UNLOCKED = 0;
+
+  // Locked onto the in-phase signal.
+  I_LOCKED = 1;
+
+  // Locked onto the quadrature signal.
+  Q_LOCKED = 2;
+
+  // Locked onto both in-phase and quadrature signal.
+  LOCKED = 3;
+}
+
+// The status of Reed-Solomon coding.
+message ReedSolomonStatus {
```

Comment:
These statuses (also `ConvolutionalCodingStatus`) make sense but I'm worried they might be too CORTEX-specific. It might be difficult to find other radios that give the same information. Is it a good approach to hold off on CORTEX-specific things until we find another radio that provide the same information?

Reply:
State we can monitor will vary by ground station but I think it's fine to include the joint set in our monitoring APIs and fill what we can - the more information for monitoring the better. Though if there are any fields you think are just never going to be useful for monitoring let me know and we should remove those.